recipes/chemdyg/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "chemdyg" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/ChemDyg/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 529e9f16cc9f6ee01a8b010c75e6546fe135921f7e63981c097087e270bec403
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - cartopy
+    - matplotlib
+    - netcdf4
+    - numpy
+    - pandas
+    - scipy
+    - xarray
+    - zppy >=2.2.0
+
+test:
+  imports:
+    - chemdyg
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/E3SM-Project/ChemDyg
+  summary: 'Chemistry Diagnostics Package'
+  description: |
+    E3SM chemistry diagnostic package version1.0 (ChemDyg v1.0) is designed for
+    E3SM chemistry development. There are total 11 types of plot and 3 types of
+    table for model-to-model and model-to-observation comparison.
+
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  doc_url: https://github.com/E3SM-Project/ChemDyg
+  dev_url: https://github.com/E3SM-Project/ChemDyg
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - hsiangheleellnl
+    - tangq
```

Comment:
@hsiangheleellnl and @tangq, please respond to this message stating that you are willing to be maintainers of this conda package (this is required as part of the review of the recipe).

Reply:
Yes, I am willing to be a maintainer of the ChemDyg package.

Reply:
Me too.